src/extensions/step_handlers/script_handler/src/script_handler.cpp

```diff
@@ -22,6 +22,8 @@
 #include <vector>
 
 #define HANDLER_PROPERTIES_SCRIPT_FILENAME "scriptFileName"
+#define HANDLER_PROPERTIES_API_VERSION "apiVersion"
+#define HANDLER_ARG_ACTION "--action"
```

Comment:
these are breaking changes, and it looks like you've added an apiVersion, but what does this mean for someone who has already written a script (*) -- the new agent would break their script.

(*) there probably aren't any, so it's likely okay for now?

Reply:
Not breaking changes. Just adding new features. Customer will get old behavior by default.